### PR TITLE
Feat: Added alert icon to component library.

### PR DESCRIPTION
**Pull request by abhijeetekad**

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    width: auto;
+    height: 354px;
+    border: 0;
+    transform: scale(1);
+    overflow: hidden;
+  }
+  section p {
+    margin-left: 1px;
+    margin-right: 1px;
+    margin-bottom: 30px;
```
Comment (KanekarSnehal) [inline components/alert.css:194]: Suggestion : please try to use relative units.

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    width: 9px;
+    background-color: aliceblue;
+    /* line-height: 3rem; */
+    /* padding-left: 1rem; */
```
Comment (KanekarSnehal) [inline components/alert.css:198]: Suggestion: remove the commented lines if no longer needed.

File: components/alert.css (hunk 1 above)
Comment (abhijeetekad) [reply components/alert.css:190]: Ok. From now on will use relative units.. Thanks for suggestion.

Comment (prathmeshjagtap) [reply components/alert.css:190]: if it is less than 10 px we can use absolute  units

**Pull request merged**